shared/src/main/scala/coop/rchain/store/KeyValueTypedStoreSyntax.scala

```diff
@@ -14,8 +14,16 @@ final class KeyValueTypedStoreOps[F[_], K, V](
     // KeyValueTypedStore extensions / syntax
     private val store: KeyValueTypedStore[F, K, V]
 ) extends AnyVal {
+  def errKVStoreExpectValue = "Error when reading from KeyValueStore: value not found."
```

Comment:
Please provide more useful info on this error message. At least include the `key` value.